### PR TITLE
feat(workbench): add Jest configuration and initial tests for LogField component

### PR DESCRIPTION
**Pull request by andersonleal**
https://linear.app/motia/issue/MOT-334/cannot-convert-undefined-or-null-to-object-when-opening-log
- Introduced Jest configuration in `jest.config.js` for testing setup.
- Added test scripts in `package.json` for running tests.
- Created `setupTests.ts` to include custom Jest matchers from `@testing-library/jest-dom`.
- Implemented comprehensive tests for `LogField` component covering various value types (string, React nodes, objects, etc.) and edge cases.

### CODE REVIEW

File: packages/workbench/src/components/logs/log-field.test.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import { render, screen } from '@testing-library/react'
+import { LogField } from './log-field'
+
+jest.mock('../../lib/utils', () => ({
```
Comment (sergiofilhowz) [inline packages/workbench/src/components/logs/log-field.test.tsx:4]: make sure these tests files won't be in the published artifacts

**Review by sergiofilhowz — APPROVED** [review]
Nice! It'll be great to have unit-tests on workbench

**Review by sergiofilhowz — APPROVED** [review]
